### ISSUE TITLE
Implement interactive mode for RolloutMigrate

### ISSUE THREAD

**Issue by teosarca** [metasfresh/metasfresh#1107]
### Is this a bug or feature request?
feature

### What is the current behavior?
when doing a manual rollout, you have to run:
> ./sql_remote.sh

If one of the migration scripts fails, the error is reported but the ./sql_remote.sh stops.

You have to fix or ignore (./sql_remote -r -f ..) the script and then run ./sql_remote.sh again.

### What is the expected or desired behavior?
Introduce a new command line parameter, for example "--ask". If the parameter is set and if there is an error while running the script, in console you shall be asked what to do next:
* Ignore - skip the migration script but record it as skipped
* Retry - retry executing the script
* Fail - stop the ./sql_remote.sh

Basically we need an IScriptsApplierListener implementation similar with de.metas.migration.applier.impl.SwingUIScriptsApplierListener but for console.

**Comment by teosarca** [comment]
Implemented as follows:

![image](https://cloud.githubusercontent.com/assets/1244701/23718897/f6e18b22-0441-11e7-97f6-4c6d5c448821.png)

**Comment by teosarca** [comment]
tested. works.
